modules/examples/todolist/src/main/scala/todo/http/TodoListApp.scala

```diff
@@ -21,69 +21,58 @@ import com.twitter.finagle.http.{Request, Response}
 import com.twitter.finagle.{Http, ListeningServer, Service}
 import com.twitter.server.TwitterServer
 import com.twitter.util.{Await, Future}
-import freestyle.free._
-import freestyle.free.implicits._
 import io.circe.generic.auto._
 import io.finch.circe._
-import todo.http.apis.{Api, AppApi}
-import runtime.implicits._
-import cats.implicits._
-import cats.~>
-import freestyle.free.config.ConfigM
-import freestyle.free.effects.error.implicits._
-import freestyle.free.config.implicits._
-import freestyle.free.loggingJVM.implicits._
-import freestyle.free.http.finch._
-import freestyle.free.logging.LoggingM
-import todo.persistence.{TagRepository, _}
-import todo.services.{AppServices, Services, TagService}
+import cats.effect.IO
+import cats.{~>, Monad}
+import cats.Monad.ops._
+import freestyle.tagless.module
+import freestyle.tagless.logging.LoggingM
+import freestyle.tagless.loggingJVM.log4s.implicits._
+import freestyle.tagless.config.ConfigM
+import freestyle.tagless.config.implicits._
+import freestyle.tagless.effects.error.ErrorM
+import freestyle.tagless.effects.error.implicits._
+import todo.http.apis.Api
+import todo.persistence.Persistence
+import todo.services.Services
 
-/**
- * Top level module including all algebra dependencies nested in other contains modules.
- * Coproduct of these algebras are calculated recursively so the entire graph is considered
- * for implicit resolution when running .interpret.
- *
- * Freestyle looks up all implicit handlers and attempts to conform a Coproduct of algebras
- * provided individual evidences for each handler exist
- */
 @module
-trait App {
-  val persistence: Persistence
-  val services: Services
+trait App[F[_]] {
+  val persistence: Persistence[F]
+  val services: Services[F]
 }
 
 object TodoListApp extends TwitterServer {
 
-  def bootstrap[F[_]](
+  import todo.runtime.implicits._
+  import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
This import should be placed in the same "runtime implicits" object, as _implicit val_.